Cubical/ZCohomology/Properties.agda

```diff
@@ -0,0 +1,64 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.ZCohomology.Properties where
+
+
+open import Cubical.ZCohomology.Base
+open import Cubical.HITs.Sn
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Pointed
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.GroupoidLaws
+open import Cubical.Data.NatMinusTwo.Base renaming (-1+_ to -1+₋₂_ ; 1+_ to 1+₋₂_)
+open import Cubical.HITs.Susp
+open import Cubical.HITs.SetTruncation 
+open import Cubical.HITs.Nullification
+open import Cubical.Data.Int
+open import Cubical.Data.Nat
+open import Cubical.HITs.Truncation
+
+open import Cubical.HITs.Pushout
+open import Cubical.Data.Sum.Base
+open import Cubical.Data.HomotopyGroup
+private
+  variable
+    ℓ : Level
+    A : Type ℓ
+
+
+
+{- Equivalence between cohomology and the reduced version -}
```

Comment:
I would suggest something like "Equivalence between cohomology of A and reduced cohomology of (A + 1)" instead (and maybe a different name for the theorem), given that there is another "equivalence between cohomology and the reduced version", namely the fact that they are equivalent for n>0.